packages/spor-react/src/datepicker/DatePicker.tsx

```diff
@@ -84,18 +75,15 @@ export const DatePicker = forwardRef<HTMLDivElement, DatePickerProps>(
       ref as React.MutableRefObject<HTMLDivElement>
     );
 
-    const styles = useMultiStyleConfig("Datepicker", {});
+    const styles = useMultiStyleConfig("Datepicker", { variant });
     const locale = useCurrentLocale();
 
-    const responsiveVariant =
-      useBreakpointValue(typeof variant === "string" ? [variant] : variant) ??
-      "simple";
-    const hasTrigger = responsiveVariant === "with-trigger";
+    // if (variant == "with-trigger" || "simple") {
+      
+    // }
```

Comment:
This should probably be uncommented and change the variant to base, right?

Reply:
not necessary, uses the values of base if the variant does not exist